server-client/index.test.ts

```diff
@@ -220,12 +220,19 @@ it('reports about connection', () => {
   expect(fired).toEqual(['1'])
 })
 
-it('removes itself on destroy', async () => {
+it('emits logux/unsubscribe action for every subscription channel on destroy', async () => {
   let test = createReporter()
-  let fired: string[] = []
+  let clientKeysDisconnectEventWasFiredFor: string[] = []
```

Comment:
Let’s make it shorter. For instance, `disconnectedKeys`

Reply:
Done